package-lock.json

```diff
@@ -14360,6 +14337,20 @@
         "node": ">=8"
       }
     },
+    "node_modules/readable-stream": {
```

Comment:
I don't think this file should have been modified since no new dependency was added or removed? Do you think you can revert these changes?

Reply:
Yeap. Sure.